plugins/scm.js

```diff
@@ -168,6 +168,20 @@ const GET_BRANCH_LIST = Joi.object().keys({
     scmContext
 }).required();
 
+const OPEN_PR = Joi.object().keys({
+    checkoutUrl,
+    token,
+    files: Joi.array().items(
+        Joi.object().keys({
+            name: Joi.string().required(),
+            content: Joi.string().required()
+        })
+    ).min(1).required(),
+    scmContext,
+    title: Joi.string().required(),
+    message: Joi.string().required()
+}).required();
```

Comment:
I can understand the attributes of `OPEN_PR` are required, but why is `OPEN_PR` itself is `required()`? Would it be okay to use `optional()` instead?

Reply:
@tkyi might know better for `data-schema` changes.

Reply:
I've spoken with Tiffany offline, and confirmed it ought not to be breaking anything. Since this `OPEN_PR` will be used for checking params in
https://github.com/screwdriver-cd/screwdriver/pull/2075/files#diff-b2fdd2f3a4a83b3e213a4257e3ca15fdR57, and will not be stored in DB and cause other effects.

Approved.